userguide/sharing.rst

```diff
@@ -1057,9 +1057,9 @@ provides more details for each configurable option.
    |                              |               |          | file) are created; existing files are not affected                                                          |
    |                              |               |          |                                                                                                             |
    +------------------------------+---------------+----------+-------------------------------------------------------------------------------------------------------------+
-   | Allow Guest Access           | checkbox      |          | if checked, no password is required to connect to the share and all users share the permissions of the      |
-   |                              |               |          | guest user defined in the :ref:`SMB` service                                                                |
-   |                              |               |          |                                                                                                             |
+   | Allow Guest Access           | checkbox      |          | if checked, no password is required to connect to the share. User login attempts with a bad password are    |
```

Comment:
See note above about sentences in tables.  "No password is required" is less clear than "a password is not required".  "User login attempts" is not clear. A user does not log in to a share, they connect to it.